src/config/index.ts

```diff
@@ -219,3 +219,8 @@ export const GOCD_PAUSED_PIPELINE_REMINDERS: GoCDPausedPipelineReminder[] = [
     notifyAfter: moment.duration(1.5, 'hour'),
   },
 ];
+
+/**
+ * GoCD Webhook Secret
+ */
+export const GOCD_WEBHOOK_SECRET = process.env.GOCD_WEBHOOK_SECRET || '';
```

Comment:
We shouldn't default to an empty string here. Startup should fail if the value is not set, otherwise we will silently be hashing things with an empty string (which is very predictable).